File: src/main/java/hacker/solution/MapDemo.java
package hacker.solution;

import java.util.HashMap;
import java.util.Map;
import java.util.Scanner;

public class MapDemo {
    public static void main(String [] args) {
        Scanner scan = new Scanner(System.in);
        int n = scan.nextInt();
        scan.nextLine();
        Map<String, Integer> phoneBook = new HashMap<String, Integer>();
        while (n-- > 0) {
            String name = scan.nextLine();
            int number = scan.nextInt();
            phoneBook.put(name,number);
            scan.nextLine();
        }
        while (scan.hasNext()) {
            String wanted = scan.nextLine();
            Integer lookup = phoneBook.get(wanted);
            if (lookup != null) {
                System.out.println(wanted+"="+lookup);
            } else System.out.println("Not found");
        }
        scan.close();
    }
}

File: src/main/kotlin/hacker/solutionk/MapDemo.kt
package hacker.solutionk

import java.util.*

object MapDemo {
    @JvmStatic
    fun main(args: Array<String>) {
        val sc = Scanner(System.`in`)
        var n = sc.nextInt();
        sc.nextLine();
        var phoneBook = HashMap<String, Integer>()
        while (n-- > 0) {
            val name = sc.nextLine()
            val phone = sc.nextInt()
            phoneBook.put(name,Integer(phone))
            sc.nextLine()
        }
        while (sc.hasNext()) {
            val s = sc.nextLine();
            val phone = phoneBook.get(s)
            if (phone != null) {
                println(s+"="+phone.toString())
            } else println("Not found")
        }
        sc.close();
    }
}